pkg/api/types.go

```diff
@@ -1305,8 +1305,9 @@ type Endpoints struct {
 //     a: [ 10.10.1.1:8675, 10.10.2.2:8675 ],
 //     b: [ 10.10.1.1:309, 10.10.2.2:309 ]
 type EndpointSubset struct {
-	Addresses []EndpointAddress
-	Ports     []EndpointPort
+	Addresses         []EndpointAddress
+	NotReadyAddresses []EndpointAddress
```

Comment:
Is this a one-of situation?  Is it valid to have Addresses and NotReadyAddresses at the same time in a single subset?

Comment pls.